packages/web-components/fast-foundation/src/menu-item/menu-item.template.ts

```diff
@@ -8,6 +8,7 @@ export const MenuItemTemplate = html<MenuItem>`
         aria-checked=${x => (x.role !== MenuItemRole.menuitem ? x.checked : void 0)}
         aria-disabled=${x => x.disabled}
         @keydown=${(x, c) => x.handleMenuItemKeyDown(c.event as KeyboardEvent)}
+        @focus=${(x, c) => x.handleFocus(c.event as FocusEvent)}
```

Comment:
Is this still needed? seems like the method does nothing.

Reply:
removed the file change